src/scene/graph-node.js

```diff
@@ -1099,6 +1099,17 @@ class GraphNode extends EventHandler {
     }
     // #endif
 
+    // fires an event on all children of the node
+    _fireOnHierarchy(name) {
+        if (this.fire) {
+            this.fire(name, this);
```

Comment:
the this argument is the wrong this. should be passed in as param.

Reply:
it's not being used on the receiving side, so it's not even clear what it should be - could pass nothing at all.

Reply:
are the insert and remove events documented or internal only? if they're internal then better to remove. if they're documented and public then you'll want to send the parent element as argument.

Reply:
removed as it's private and unused.